source/RialtoGStreamerMSEAudioSink.cpp

```diff
@@ -521,6 +571,44 @@ static void rialto_mse_audio_sink_set_property(GObject *object, guint propId, co
         }
         break;
     }
+    case PROP_AUDIO_FADE:
+    {
+        const gchar *audioFadeStr = g_value_get_string(value);
+        double volume = kDefaultVolume;
+        uint32_t duration = kDefaultVolumeDuration;
+        int easeTypeInt = convertEaseTypeToInt(kDefaultEaseType);
+
+        int parsedItems = sscanf(audioFadeStr, "%lf,%u,%d", &volume, &duration, &easeTypeInt);
```

Comment:
Realised that this is incorrect, your parseing a fade volume (0-100) into a double. But dont convert it correctly for setVolume()